cmake/std/PullRequestLinuxDriverMerge.py

```diff
@@ -93,43 +94,33 @@ def merge_branch(source_url, source_branch, target_branch, sourceSHA):
     if 'source_remote' in remote_list:
         print('git remote exists, removing it', file=sys.stdout)
         subprocess.check_call(['git', 'remote', 'rm', 'source_remote'])
-    subprocess.check_call(['git', 'remote', 'add', 'source_remote',
-                           source_url])
+    subprocess.check_call(['git', 'remote', 'add', 'source_remote', source_url])
 
     fetch_succeeded = False
     for i in range(3):
         try:
-            subprocess.check_call(['git', 'fetch', 'source_remote',
-                                   source_branch])
+            subprocess.check_call(['git', 'fetch', 'source_remote', source_branch])
             fetch_succeeded = True
             break
         except subprocess.CalledProcessError:
             pass
     if not fetch_succeeded:
         raise SystemExit(12)
 
-    subprocess.check_call(['git', 'fetch', 'origin',
-                           target_branch])
-    subprocess.check_call(['git', 'reset', '--hard',
-                           'HEAD'])
-    subprocess.check_call(['git', 'checkout', '-B',
-                           target_branch, 'origin/' + target_branch])
-    subprocess.check_call(['git', 'merge', '--no-edit',
-                           'source_remote/' + source_branch]),
+    subprocess.check_call(['git', 'fetch', 'origin', target_branch])
+    subprocess.check_call(['git', 'reset', '--hard', 'HEAD'])
+    subprocess.check_call(['git', 'checkout', '-B', target_branch, 'origin/' + target_branch])
+    subprocess.check_call(['git', 'merge', '--no-edit', 'source_remote/' + source_branch]),
 
-    actual_source_SHA = subprocess.check_output(['git', 'rev-parse',
-                                                 'source_remote/' + source_branch])
+    actual_source_SHA = subprocess.check_output(['git', 'rev-parse', 'source_remote/' + source_branch])
 
     actual_source_SHA = actual_source_SHA.strip()
 
     if actual_source_SHA != sourceSHA:
         print('The SHA ({source_sha}) for the last commit on branch {source_branch}'.format(source_sha=actual_source_SHA,
-                                                                                            source_branch=source_branch),
-              file=sys.stdout)
-        print('  in repo {source_repo} is different than the expected SHA,'.format(source_repo=source_url),
-              file=sys.stdout)
-        print('  which is: {source_sha}.'.format(source_sha=sourceSHA),
-              file=sys.stdout)
+                                                                                            source_branch=source_branch))
+        print('  in repo {source_repo} is different than the expected SHA,'.format(source_repo=source_url))
```

Comment:
Actually these are errors so the original should have been sys.stderr

Reply:
I'll re-add the `file=sys.stdout` for now as a placeholder. We can change these to `sys.stderr` sometime if we want but we'll need to make sure the right information still gets captured in the logs that the autotester prints out as well as update the unit testing of the scripts to reflect the change.